config.toml.example

```diff
@@ -9,4 +9,5 @@ BBNChainID = "euphrates-0.4.0"
 BBNRPCAddress = "https://rpc-euphrates.devnet.babylonchain.io"
 GRPCListener = "0.0.0.0:50051"
 HTTPListener = "0.0.0.0:8080"
-PollInterval = "10s"
\ No newline at end of file
+PollInterval = "10s"
+BatchSize = 10
```

Comment:
is this only used for fast sync? if that's the case, calling it FastSyncBatchSize would be better

Reply:
discussed offline. this is not really fast sync

Reply:
yep, to disable batch polling / inserts, consumer can set `BatchSize` to 1